estate/models/estate_property.py

```diff
@@ -0,0 +1,45 @@
+from odoo import fields, models
+from dateutil.relativedelta import relativedelta
+
+
+class Property(models.Model):
+    _name = "estate.property"
+    _description = "All properties"
+
+    name = fields.Char(required=True)
+    description = fields.Text()
+    postcode = fields.Char()
+    date_availability = fields.Date(
+        copy=False,
+        default=fields.Datetime.today() + relativedelta(months=+3),
```

Comment:
```suggestion
        default=fields.Date.add(fields.Date.today(), months=3),
```
1. Your code is (almost, see 2) correct, just know that we have some built-in helper for that.
It allows to avoid the import of relativedelta :)
2. You should use `fields.Date.today()` instead of Datetime since you have a `Date' type field. Otherwise you trigger an extra cast needlessly.

Reply:
Thanks for that. I was looking into the helper but missed this `fields.Date.add()`. This is now updated.